### PR TITLE
Tests for different project parts

### PR DESCRIPTION
**Pull request by alf3ratz**
Merge tests for tree-sitter, enry and dulwich

### CODE REVIEW

File: test/enry_test.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from info_processing.code_processing.enry_processor import get_language
+
+from pathlib2 import Path
+
+
+class EnryTest(unittest.TestCase):
+    def test_get_language(self):
+        path = str(Path(f"{Path().cwd}/enry_test.py"))
+        self.assertEqual("python", get_language(path))
+        print()
```
Comment (EgorBu) [inline test/enry_test.py]:
```suggestion
```

File: test/enry_test.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from info_processing.code_processing.enry_processor import get_language
+
+from pathlib2 import Path
+
+
+class EnryTest(unittest.TestCase):
+    def test_get_language(self):
+        path = str(Path(f"{Path().cwd}/enry_test.py"))
```
Comment (EgorBu) [inline test/enry_test.py:10]: you can check java file too

File: test/git_test.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import logging
+import shutil
+import unittest
+
+from info_processing.git.helpers import get_repository_info
```
Comment (EgorBu) [inline test/git_test.py:5]: wrong order of imports. It should be:
* standard library
* external dependencies
* from this module

**Review by EgorBu — CHANGES_REQUESTED** [review]
Almost done!

File: test/git_test.py (hunk 3 above)
Comment (alf3ratz) [reply test/git_test.py:5]: linter has another opinion about that

Comment (EgorBu) [reply test/git_test.py:5]: what does it say?

Comment (EgorBu) [reply test/git_test.py:5]: https://peps.python.org/pep-0008/#:~:text=Imports%20are%20always%20put%20at,Related%20third%20party%20imports. - anyway - let's follow pep8 style guide

File: test/git_test.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+from pathlib2 import Path
```
Comment (EgorBu) [inline test/git_test.py:7]:
```suggestion
import logging
import shutil
import unittest

from pathlib2 import Path

from info_processing.git.helpers import get_repository_info
```

**Review by EgorBu — APPROVED** [review]

**Pull request merged**